tools/kafka/loopback/README.md

```diff
@@ -1,53 +1,86 @@
-# Loopback service for kafka
+# Loopback service for Kafka
 
-This service is a helper for kafka. It's goal is to transform messages sent from devices to dojot into messages sent from dojot to devices. It transfers incoming messages from **device-data** topic to **device-manager** topic adapting them according to dojot's message schema.
+This service's goal is to simulate actuations in dojot by forwarding messages in the `device-data`
+topic (publishing topic) to the `device-manager.device` topic (actuation topic), adapting its format
+to be dojot-compliant.
 
 # Configurations
+
 ## Environment Variables
 
-Key                      | Purpose                                                             | Default Value   			| Valid Values      |
------------------------- | ------------------------------------------------------------------- | -------------------------- | ----------------- |
-DOJOT_USERNAME           | username to access dojot                 						   | admin           			| string   		    |
-DOJOT_PASSWORD           | password to access dojot                 						   | admin           			| string   		    |
-AUTH_ADDRESS             | Address of the auth service                                         | http://auth:5000           | hostname/IP:port  |
-DATA_BROKER_ADDRESS      | Address of the data broker service                                  | http://data-broker:80    	| hostname/IP:port  |
-KAFKA_BROKER_LIST        | Addresses of the kafka brokers separated by a comma                 | kafka-server:9092			| hostname/IP:port  |
-LOOPBACK_CONSUMER_GROUP  | Kafka consumer group                                                | loopback-group             | string            |
-DEVICE_DATA_TOPIC    	 | Topic to consume from                                               | device-data        		| string            |
-DEVICE_MANAGER_TOPIC     | Topic to produce to                                                 | dojot.device-manager.device| string            |
+Key                     | Purpose                               | Default Value               | Valid Values     |
+----------------------- | ------------------------------------- | --------------------------- | ---------------- |
+AUTH_ADDRESS            | Address of the Auth service           | http://auth:5000            | hostname/IP:port |
+DATA_BROKER_ADDRESS     | Address of the Data Broker service    | http://data-broker:80       | hostname/IP:port |
+DEVICE_DATA_TOPIC       | Topic to be consumed                  | device-data                 | string           |
+DEVICE_MANAGER_TOPIC    | Topic to produce to                   | dojot.device-manager.device | string           |
+DOJOT_PASSWORD          | dojot's user password                 | admin                       | string           |
+DOJOT_USERNAME          | dojot's user name                     | admin                       | string           |
```

Comment:
Is this necessary?

Reply:
By looking at the code, it is used, but it doesn't really make sense, since this is an internal component that shouldn't make requests to the APIGW... maybe we could open a ticket to solve this later on.

Reply:
I think it's a good idea to open a ticket. As you said, it doesn't make sense for an internal component.

Reply:
Actually, Data Broker has its own authentication system that requires the JWT to be passed, so we need it.

Reply:
I don't understand why we need the data-broker since we changed the way the Kafka's topic are named. In my opinion, we should try to remove this dependency, passing the topics explicit. What do you think?

Reply:
It makes sense. So we could pass this PR and then make this change in another one? What do you think?

Reply:
I agree with you. Let's merge this PR and open a new one with the change.